packages/common/src/core/schemas/internal/getEditorSchemas.ts

```diff
@@ -68,7 +74,7 @@ export async function getEntityEditorSchemas(
       }[type as SiteEditorType]();
       uiSchema = await siteModule.buildUiSchema(
         i18nScope,
-        entity as IHubSite,
+        options as IHubSite,
```

Comment:
I'm wondering if this should now be `options as IEntityEditorOptions` and all of the corresponding `buildUiSchema` functions should also update their functional interface to expect an `IEntityEditorOptions`...?